apiserver/facades/client/client/status.go

```diff
@@ -374,11 +389,19 @@ type statusContext struct {
 	applications map[string]*state.Application
 
 	// remote applications: application name -> application
-	remoteApplications map[string]*state.RemoteApplication
-	relations          map[string][]*state.Relation
-	units              map[string]map[string]*state.Unit
-	latestCharms       map[charm.URL]*state.Charm
-	leaders            map[string]string
+	consumerRemoteApplications map[string]*state.RemoteApplication
```

Comment:
I don't understand what `consumerRemoteApplications` means. Or rather, I can see two possible opposite meanings. Is it remote applications that represent the consumer (ie we're on the offering side of the CMR)? Or remote applications that consume an offer (so we're on the consuming side, and the remote application is a proxy for an offered application)?

(I have the same problem with `isConsumerProxy` actually.)